ext/SymbolicsGroebnerExt.jl

```diff
@@ -70,4 +70,97 @@ function Symbolics.is_groebner_basis(polynomials::Vector{Num}; kwargs...)
     Groebner.isgroebner(polynoms; kwargs...)
 end
 
+function Symbolics.solve_multivar(eqs::Vector{Num}, vars::Vector{Num}, mult=false)
+    # do the trick
```

Comment:
`HAT` is sometimes called "a separating form". You can put a comment in the code with a reference; one classic reference is

> Rouillier, F. Solving Zero-Dimensional Systems Through the Rational Univariate Representation. AAECC 9, 433–461 (1999). https://doi.org/10.1007/s002000050114